backend/addcorpus/corpus.py

```diff
@@ -309,7 +328,7 @@ def source2dicts(self, source):
         default implementation for XML layouts; may be subclassed if more
         '''
         # Make sure that extractors are sensible
-        self._reject_extractors(extract.HTML, extract.CSV)
+        self._reject_extractors(extract.CSV)
```

Comment:
Ah, good catch - this may be a remainder of an experiment that I forgot to remove.

Reply:
Nope -- it turns out that PeacePortal corpora need the attribute filter that is specified on the HTML extractor. Inheriting from HTMLCorpusDefinition *and* XMLCorpusDefinition also doesn't solve this problem. On closer inspection, it turns out that the HTML filter is not really designed for HTML files, but is a subclass of the XML extractor which overrides the `_select` method to select by attribute name/value, rather than only tags. It's not used anywhere outside the PeacePortal corpora, so I will rename to avoid this confusion in the future.